fatiando/tests/mesher/test_prism_mesh.py

```diff
@@ -0,0 +1,11 @@
+from ...mesher import PrismMesh
+
+
+def test_prism_mesh_copy():
+    p1 = PrismMesh((0, 1, 0, 2, 0, 3), (1, 2, 2))
+    p1.addprop('density', 3200)
```

Comment:
For the mesh classes in Fatiando, the physical properties are arrays of values (one for each cell in the mesh). Could you test this using arrays? You can create the array with something like `p1.addprop('density', 3200 + np.zeros(p1.size))`